rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.h

```diff
@@ -0,0 +1,84 @@
+﻿#pragma once
+
+#include "overlays.h"
+
+namespace rsx
+{
+	namespace overlays
+	{
+		struct perf_metrics_overlay : overlay
+		{
+		private:
+			/*
+			   minimal - fps
+			   low - fps, total cpu usage
+			   medium - fps, detailed cpu usage
+			   high - fps, frametime, detailed cpu usage, thread number, rsx load
+			 */
+			detail_level m_detail;
+		
+			screen_quadrant m_quadrant;
+			positioni m_position;
+		
+			label m_body;
+			label m_titles;
+		
+			CPUStats m_cpu_stats;
+			Timer m_update_timer;
+			u32 m_update_interval; // in ms
+			u32 m_frames{ 0 };
+			std::string m_font;
+			u32 m_font_size;
+			u32 m_margin_x; // horizontal distance to the screen border relative to the screen_quadrant in px
+			u32 m_margin_y; // vertical distance to the screen border relative to the screen_quadrant in px
+			f32 m_opacity;  // 0..1
+		
+			bool m_center_x{ false }; // center the overlay horizontally
+			bool m_center_y{ false }; // center the overlay vertically
+		
+			std::string m_color_body;
+			std::string m_background_body;
+		
+			std::string m_color_title;
+			std::string m_background_title;
+		
+			bool m_force_update;
+			bool m_is_initialised{ false };
+		
+			const std::string title1_medium{ "CPU Utilization:" };
+			const std::string title1_high{ "Host Utilization (CPU):" };
+			const std::string title2{ "Guest Utilization (PS3):" };
```

Comment:
Since they're const, can these be `const std::string_view` instead?

Reply:
I don't wanna change anything apart from my additions

Reply:
Well, those do show as additions, but as I understand this is only **moved** code. Fair enough.